### PR TITLE
move watchCache metrics to a seperate file

### PR DESCRIPTION
**Pull request by gongguan**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
pre work of [#90091](https://github.com/kubernetes/kubernetes/pull/90091/files)

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by gongguan** [comment]
https://github.com/kubernetes/kubernetes/pull/90091/files#r409069822
Sorry I don't know how to set two values(old and new capacity) in one metric counter.

**Comment by gongguan** [comment]
> For [#90091 (comment)](https://github.com/kubernetes/kubernetes/pull/90091#discussion_r409069822)
> Sorry I don't know how to set two values(old and new capacity) in one metric counter.

Um... You mean handle two counters(increase and decrease) in one function according to old and new capacity?

File: staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go [hunk 1]
```diff
@@ -62,6 +62,22 @@ var (
 		},
 		[]string{"resource"},
 	)
+	watchCacheCapacityIncreaseTotal = metrics.NewCounterVec(
```
Comment (wojtek-t) [inline staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go:65]: Sorry - apparently I wasn't clear what I meant.
What I had on my mind was:
- in this PR simply create a new metrics.go file and move the existing initCounter metric to that file
- what you add in this file, move to your original PR, though add it to this newly created metrics.go file

Does that make sense?

Comment (gongguan) [reply staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go]: @wojtek-t  Sorry for misunderstanding it. Updated, PTAL

Comment (wojtek-t) [reply staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go]: No need to sorry - thanks a lot for updating!

**Pull request merged**